src/client/pages/Debugger/components/Offer.tsx

```diff
@@ -124,371 +197,90 @@ export const Offer: React.FC = () => {
+                      )}
+                      {quoteType === QuoteType.NorwegianHome && (
+                        <NorwegianHome formik={props} />
+                      )}
+                      {quoteType === QuoteType.NorwegianTravel && (
                         <NorwegianTravel formik={props} />
-                      </QuoteForm>
-                    </>
-                  )}
-                </Formik>
-              )}
-
-              {quoteType === QuoteType.SwedishApartment && (
-                <Formik<Partial<CreateQuoteInput>>
-                  initialValues={{
-                    firstName: 'Blargh',
-                    lastName: 'Blarghson',
-                    currentInsurer: '',
-                    birthDate: '1995-09-29',
-                    ssn: '',
-                    startDate: '',
-                    email: 'blargis@hedvig.com',
-                    swedishApartment: {
-                      street: 'Storgatan 1',
-                      zipCode: '',
-                      livingSpace: 23,
-                      householdSize: 1,
-                      type: ApartmentType.Rent,
-                    },
-                  }}
-                  onSubmit={async (values) => {
-                    await createQuote(
-                      storage,
-                      localeIsoCode,
-                    )({
-                      input: {
-                        ...values,
-                        id: quoteId,
-                        currentInsurer: values.currentInsurer || undefined,
-                        // @ts-ignore
-                        startDate: values.startDate || undefined,
-                      },
-                    })
+                      )}
+                      {quoteType === QuoteType.DanishHome && (
+                        <DanishHome formik={props} />
+                      )}
 
-                    if (refetch) {
-                      await refetch()
-                    }
-                  }}
-                >
-                  {(props) => (
-                    <>
-                      <QuoteForm formik={props}>
-                        <SwedishApartment formik={props} />
-                      </QuoteForm>
-                    </>
-                  )}
-                </Formik>
-              )}
+                      <Button
+                        background={colorsV3.purple500}
+                        foreground={colorsV3.gray900}
+                        type="submit"
+                      >
+                        Create quote
+                      </Button>
+                    </Form>
+                  </>
+                )}
+              </Formik>
             </>
           )}
           {data?.quote && <pre>{JSON.stringify(data, null, 2)}</pre>}
+          {quoteCreatingError && (
+            <>
+              <h2>Something went wrong 😔</h2>
+              <p>
+                <u>Error message:</u> {quoteCreatingError}
+              </p>
```

Comment:
Might be easier to read in a pre tag :) 
```suggestion
              <p>Error message:</p> 
               <pre>{quoteCreatingError}</pre>
```

Reply:
It actually gets worse then since the `<pre>` tag makes it use the line breaks from the original formatting - which means _no_ line breaks.

Reply:
![image](https://user-images.githubusercontent.com/42962286/97276810-fb1a3a00-1837-11eb-8dd8-01ea65f78e22.png)

Reply:
I thought it was easier to read when you got a longer error massage actually 😁  But let's go with a paragraph 👍
![Screenshot 2020-10-27 at 09 41 35](https://user-images.githubusercontent.com/6661511/97277576-ed18e900-1838-11eb-8acf-d494dd50cabd.png)

Reply:
Haha WOW never gotten that error message 😅 What did you do to trigger that?

Reply:
And YES, you're of course right it's easier to read those really long error messages with pre tag - I've just been so focused on the errors from submitting wrong id so didn't check this 😳

Reply:
OK, the best solution was the `<pre>` tag with `whiteSpace: 'pre-wrap'` styling to make it not overflow:
![image](https://user-images.githubusercontent.com/42962286/97282031-7aab0780-183e-11eb-8b5f-6e175ea3acf6.png)
Thank you mister for being so persistent! 🙏

Reply:
Snyggt! 👌  Haha, sorry for being a nag 😆